### PR TITLE
Bugfix/general fixes

### LINKED ISSUE #129: [BUG] Balance is empty on zero balance
When a users balance is zero, it is shown as an empty card.

### LINKED ISSUE #137: [BUG] top up without filling anything in give NaN
Steps:
1. go to top up
2. Fill in no number
3. Modal appear with NaN

### LINKED ISSUE #138: [BUG] Navigating to transactions does not work on mobile
Steps:
1. Be on mobile (or web emulator)
2. go to any page that is not transaction
3. Try to navigate to transactions

### LINKED ISSUE #139: [BUG] Topping cannot secretely change to 10 euro
Despite it being very funny, we cannot have the top up secretly change to 10 euro when putting in a value below 10 euros.

### LINKED ISSUE #143: [BUG] RBAC is not checked for showing `delete` button on transaction
Closely releated to GH-44 

Could not find outstanding issues regarding RBAC, permissions, rights or ACLs

Users see the delete button even if they don´t have the permission to delete a transaction. Verified that the backend properly does a 403 (if we ignore the CORS failures)

### LINKED ISSUE #144: Google Fonts are hosted externally
https://github.com/GEWIS/sudosos-frontend-vue3/blob/e895f587c42ca9e85c02405abbf56e8a7bd8c06d/src/styles/themes/_fonts.scss#L1-L3

Bad for privacy reasons, German court ruled in 2022 that this is a violation of GDPR because Google still receives IP addresses while the user has not consented to this. With gewis.nl we opted to host the fonts ourselves (and will completely move away from Google Fonts in the future).

### LINKED ISSUE #145: [BUG] Waived fines not shown correctly
Waived fines are somehow shown as a negative SudoSOS topup.
image
image

### LINKED ISSUE #146: [BUG] Invoices are not shown correctly
See this
image
image

### LINKED ISSUE #147: [BUG] Can't create users
Whenever I try adding a user, the checkboxes give errors.

### PR DESCRIPTION
**Pull request by CodeNamedRobin**
Fixes #129.
Fixes #137.
Fixes #138.
Fixes #139.
Fixes #143.
Fixes #144.
Fixes #145.
Fixes #146.
Fixes #147.

### CODE REVIEW

**Comment by SuperVK** [comment]
If I am in the minus, I cannot top up such that I become 0 again.

File: src/components/BalanceComponent.vue [hunk 1]
```diff
@@ -42,6 +42,7 @@ const updateUserBalance = async () => {
   if (props.user) {
     const response = await apiService.balance.getBalanceId(props.user.id);
     userBalance.value = response.data;
+    console.error(userBalance.value);
```
Comment (SuperVK) [inline src/components/BalanceComponent.vue]: Should this be in here?

File: src/components/BalanceComponent.vue [hunk 2]
```diff
@@ -51,6 +52,7 @@ const updateUserBalance = async () => {
     }
     await userStore.fetchCurrentUserBalance(auth.getUser.id, apiService);
     userBalance.value = userStore.getCurrentUser.balance;
+    console.error(userBalance.value);
```
Comment (SuperVK) [inline src/components/BalanceComponent.vue]: Idem

**Comment by SuperVK** [comment]
Looks all goed besides that console.errors, I also deleted all the fonts that we were never importing when using the URL import. I think it is fine to merge!

File: src/components/BalanceComponent.vue (hunk 1 above)
Comment (CodeNamedRobin) [reply src/components/BalanceComponent.vue]: Nope, forgot to remove.

File: src/components/BalanceComponent.vue (hunk 2 above)
Comment (CodeNamedRobin) [reply src/components/BalanceComponent.vue]: Idem

**Review by SuperVK — APPROVED** [review]
LGTM

**Pull request merged**